examples/webhook-signing/typescript-node-express/express-ts.ts

```diff
@@ -0,0 +1,76 @@
+import Stripe from 'stripe';
+import express from 'express';
+import bodyParser from 'body-parser';
+import env from 'dotenv';
+
+env.config();
+
+const stripe: Stripe = new Stripe(process.env.STRIPE_SECRET_KEY, {
+  apiVersion: '2019-12-03',
+  typescript: true,
+});
+
+const webhookSecret: string = process.env.STRIPE_WEBHOOK_SECRET;
+
+const app: express.Application = express();
+
+// Only use the raw body parser for webhooks.
+app.use(
+  (
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ): void => {
+    if (req.originalUrl === '/webhook') {
+      next();
+    } else {
+      bodyParser.json()(req, res, next);
+    }
+  }
+);
+
+// Stripe requires the raw body to construct the event
+app.post(
+  '/webhook',
+  bodyParser.raw({type: 'application/json'}),
+  (req: express.Request, res: express.Response): void => {
+    const sig: string | string[] = req.headers['stripe-signature'];
+
+    let event: Stripe.Event;
+
+    try {
+      event = stripe.webhooks.constructEvent(req.body, sig, webhookSecret);
+    } catch (err) {
+      // On error, return the error message
+      res.status(400).send(`Webhook Error: ${err.message}`);
+      return;
+    }
+
+    // Do something with event
+    console.log('Success:', event.id);
+
+    // Cast event data to Stripe object
+    let stripeObject: Stripe.PaymentIntent | Stripe.Charge;
+
+    switch (event.type) {
+      case 'payment_intent.succeeded':
+        stripeObject = event.data.object as Stripe.PaymentIntent;
+        console.log(`PaymentIntent status: ${stripeObject.status}`);
+        break;
+      case 'charge.succeeded':
+        stripeObject = event.data.object as Stripe.Charge;
+        console.log(`Charge id: ${stripeObject.id}`);
+        break;
```

Comment:
should we add a `default` case so that if people run this with some other event type, they see something here?